product_details/__init__.py

```diff
@@ -11,7 +11,7 @@ class MissingJSONData(IOError):
 
 
 __version__ = '0.8.2'
-__all__ = ['__version__', 'product_details', 'version_compare']
+__all__ = ['__version__', 'product_details']
```

Comment:
Whoops, forgot that this slipped in, but we don't define a `version_compare` here anyway.